src/pages/AllBlogposts.js

```diff
@@ -0,0 +1,57 @@
+import React, {useState} from 'react';
+import {Link} from "react-router-dom";
+import page from "./page.module.css";
+import json from "../data/posts.json";
+import {now, useConditionalEffect} from "../helpers/utils";
+
+
+export default function AllBlogposts() {
+    const [data, setData] = useState(null);
+
+    function fetchData() {
+        console.log(now(), 'in fetchData()');
+        console.log(`json:`, json);
+        setData(json);
+    }
+
+    useConditionalEffect(fetchData, json);
+
+    return (
```

Comment:
De hele pagina had in dit geval veel simpeler gekunt door over de posts heen te mappen en vervolgens elementen aan te maken per post. Zie uitwerkingen van NOVI.